README.md

```diff
@@ -21,6 +23,23 @@ eXtreme Spacetimes (SXS) research program into the multi-messenger
 astrophysics of neutron star mergers, core-collapse supernovae, and
 gamma-ray bursts.
 
+## Citing SpECTRE
+
+Please cite SpECTRE in any publications that make use of its code or data. Cite
+the latest version that you use in your publication. The DOI for this version
+is:
+
+- DOI: [10.5281/zenodo.4290405](https://doi.org/10.5281/zenodo.4290405)
+
+You can cite this BibTeX entry in your publication:
+
+- [Find BibTeX entry for this version on Zenodo](https://zenodo.org/record/4290405/export/hx)
+
+To aid reproducibility of your scientific results with SpECTRE we recommend you
+keep track of the version(s) you used and report this information in your
+publication. Also supply the YAML input files and, if appropriate, any
```

Comment:
Add "We also recommend you supply... as supplemental material to the publication."

Reply:
Is "supplemental" and "supplementary" equally correct?